storage/wdpost/wdpost_run.go

```diff
@@ -218,7 +218,20 @@ func (s *WindowPoStScheduler) checkSectors(ctx context.Context, check bitfield.B
 		})
 	}
 
-	bad, err := s.faultTracker.CheckProvable(ctx, s.proofType, tocheck, func(ctx context.Context, id abi.SectorID) (cid.Cid, bool, error) {
+	nv, err := s.api.StateNetworkVersion(ctx, types.EmptyTSK)
+	if err != nil {
+		return bitfield.BitField{}, xerrors.Errorf("failed to get network version: %w", err)
+	}
+
+	pp := s.proofType
+	if nv >= network.Version19 {
+		pp, err = pp.ToV1_1PostProof()
+		if err != nil {
+			return bitfield.BitField{}, xerrors.Errorf("failed to convert to v1_1 post proof: %w", err)
+		}
+	}
```

Comment:
Technically we don't need a V1_1 for a proving check, but I guess it doesn't hurt too much.

Tho to reduce churn if we add new proof types, I'd swap `pp.ToV1_1PostProof()` for something that takes network version, and gives a proof type for that version

Reply:
Honestly, I just wanna drop this check after the nv19 upgrade.

Reply:
(adding a note)